### ISSUE TITLE
Increased x86 Support or Documentation

### ISSUE THREAD

**Issue by camgaertner** [cryptax/androidre#5]
Hello and thanks for the work done here thus far,

Since x86 is much faster, many will jump to use it as pointed out in the `README`. However, I found a couple issues in the image that could be useful to fix in the Dockerfile or address in documentation.
These are:
* Cannot `adb root` into the x86 emulator (aka `emulator9`) (frida requires root)
   * Internet searches show that the emulator image may need to be changed to provide root access.
* Frida-server version is ARM. 
   * Not really an "issue", but beginners may be slow to find this doesn't work with the x86 emulator. Also, there's not a mention of this in the README. Under `Using frida`, users could get stuck at `./frida-server`

I'd be happy to open a PR for updating the documentation if you like, but it would be really cool to add x86 support throughout this image too. Let me know!

Cameron

**Comment by cryptax** [comment]
Yes, you are right, there are many improvements that should be made to the x86 support. Actually, part of the docker images are now outdated... and I usually find it easier now to pop up an emulator from Android Studio, or Genymotion, than use my docker...
I'll keep this issue and work on it asap.

**Comment by ghost** [comment]
arm apk can install on x86 emulator?

**Comment by cryptax** [comment]
yes, only on recent versions (Android 11 if I am correct).
